src/ripple/nodestore/DatabaseShard.h

```diff
@@ -165,6 +171,12 @@ class DatabaseShard : public Database
         std::function<bool(soci::session& session, std::uint32_t index)> const&
             callback) = 0;
 
+    virtual bool
+    doCallForTransactionSQL(
+        std::uint32_t const shardIndex,
```

Comment:
Nit: Not necessary/customary to declare this param const.

Reply:
Done